### PR TITLE
Update vMF train config

### PR DESCRIPTION
**Pull request by brendon-boldt**

### CODE REVIEW

**Review by FelixLabelle — APPROVED** [review]
Looks good, runs at the very least

File: scripts/train.sh [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-               --input_feed\
-               --tie_embed\
-               --unk_replace\
-               --fasttext_embeds_path ~/data/corpus.fasttext.txt
+python main.py \
+    --checkpoint_path vmf \
+    --device cuda \
+    --dropout 0.0 \
+    --enc_bidirectional \
+    --enc_embed_size 300 \
```
Comment (artidoro) [inline scripts/train.sh]: I would leave enc_embed_size to default

File: scripts/train.sh [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+    --fasttext_embeds_path ~/data/corpus.fasttext.txt \
+    --input_feed \
+    --loss_function vmf \
+    --lr 5e-4 \
+    --max_len 100 \
+    --mode train \
+    --src_language de \
+    --tie_embed \
```
Comment (artidoro) [inline scripts/train.sh]: I would remove --tie_embed because it has no effect for vmf loss.

**Review by artidoro — CHANGES_REQUESTED** [review]

**Review by artidoro — APPROVED** [review]
Looks good, I would recommend also changing the name of the checkpoint_path to vmf_deen, ...

File: main.py [hunk 3]
```diff
@@ -90,6 +90,8 @@ def parse_args(args):
         # Use checkpoint arguments if required.
         if args['use_checkpoint_args']:
             checkpoint_args = checkpoint['args']
+            if args['overwrite_args'] is None:
```
Comment (artidoro) [inline main.py:93]: I think you can add a default to the argparse.

**Pull request merged**